react/README.md

```diff
@@ -1,4 +1,4 @@
-# DevExtreme React Switch - Getting Started 
+# DevExtreme React Switch - Getting Started  
 
 This repository stores the code examples of the React Switch component for the [Getting Started with Switch](https://js.devexpress.com/Documentation/Guide/UI_Components/Switch/Getting_Started_with_Switch/) tutorial. The component allows users to switch between two states: **ON** and **OFF**.
 
```

Comment:
Can you make the following changes to the lines below?

Line 40:
Please change "(webpack, Babel, ESLint, etc)" to "(webpack, Babel, ESLint, etc.)"

Line 42:
"However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it."
1) Please add a comma after "However".
2) What do you mean by "when you are ready for it"? Does "it" refer to _customization_? If so, is this last phrase necessary? For example:

"However, we understand that this tool wouldn't be useful if you couldn't customize it."

Reply:
Ok, thanks!